proto/rig/api/v1/capsule/service.proto

```diff
@@ -400,6 +402,57 @@ message DeployResponse {
   Revision revision = 4;
 }
 
+// Deploy request. This will deploy a number of changes which results in a new
+// rollout.
+message ReviewableDeployRequest {
```

Comment:
Terminology; Should we call this "Propose" instead? Or `ProposeRollout`? Or something similar. 


(comment is wrong, right?)